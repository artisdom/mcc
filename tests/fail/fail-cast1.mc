int main()
{
  int *p;
  /* Shouldn't be able to assign void * to int * without casting.*/
  p = malloc(24);
  return 0;
}
