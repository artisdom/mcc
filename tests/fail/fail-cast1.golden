Type error: expected one of [int *] but got void *. Error occured in statement:
p = malloc(24);